### PR TITLE
Add scripts to run rolling shutter locally

### PR DESCRIPTION
**Pull request by jannikluhn**

### CODE REVIEW

File: rolling-shutter/keyperimpl/optimism/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ func BootstrapValidators(config *Config) error {
 	}
 
 	ms := fx.NewRPCMessageSender(shmcl, config.SigningKey.Key)
+	ms.AllowedToSend.Store(true)
```
Comment (konradkonrad) [inline rolling-shutter/keyperimpl/optimism/bootstrap/bootstrap.go:37]: same question as in #489 : it is intentional to fix this in the `optimism` keyperimpl?

File: play/manual/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+   `contracts_root_dir`.
+4. Run `./ethereum.sh`. This will output a set of contract addresses.
+5. Copy and paste the contract addresses into the configuration files
+   `keyper-0.toml`, `keyper-1.toml`, and `keyper-2.toml` under the section
+   `[Gnosis.Contracts]`. Additionally, copy the keyper set manager address into
+   `op-bootstrap-config.toml` under `KeyperSetManager`.
+
+> Tip: A newly deployed validator registry won't have any registrations, so
+> keypers won't produce any keys. Oftentimes it is therefore helpful to use the
+> official validator registry on Chiado instead.
```
Comment (ylembachar) [inline play/manual/README.md:23]: Where can we find the official validator registry on Chiado or Gnosis?

File: play/manual/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+6. Run `./initdb.sh`
+7. Run `./p2p.sh` and keep it running.
+8. Run `./shuttermint.sh` and keep it running.
+9. Wait until the activation block of the first keyper set is reached. The time
+   it takes is defined by `activation_delta` defined in `ethereum.sh`.
+10. Run the keypers with `./k.sh 0`, `./k.sh 1`, and `./k.sh 2`. Optionally, set
+    the `ROLLING_SHUTTER_LOGLEVEL` environment variable to
+    `:debug,dht:info,pubsub:info,swarm:info` to get reasonable log outputs.
```
Comment (ylembachar) [inline play/manual/README.md:32]: Should we add here some logs we should expect to see if all is running correctly?

File: play/manual/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Manual Setup
+
+This directory contains a few scripts and corresponding config file templates to
+semi-manually setup and run a set of keypers locally. Basic instructions:
+
+0. Build `rolling-shutter` with `make -C ../../rolling-shutter build`.
+1. Run a postgres database, e.g. with `process-compose up` in `/play`
+2. Set the variable `private_key` in `ethereum.sh` to a private key with funds
+   on Chiado.
+3. Clone the repos
```
Comment (ylembachar) [inline play/manual/README.md:10]: Could you also add here that we need to extract submodules for these two repos? `git submodule update --init --recursive` thanks

File: play/manual/README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Manual Setup
+
+This directory contains a few scripts and corresponding config file templates to
+semi-manually setup and run a set of keypers locally. Basic instructions:
+
+0. Build `rolling-shutter` with `make -C ../../rolling-shutter build`.
+1. Run a postgres database, e.g. with `process-compose up` in `/play`
```
Comment (ylembachar) [inline play/manual/README.md:7]: Could you please also add the db directory initialization phase? I had to run `initdb -D work/db/` to successfully run `process-compose up`

File: play/manual/rolling-shutter [hunk 6]
```diff
@@ -0,0 +1 @@
+/Users/jannik/Projects/Brainbot/rolling-shutter/play/manual/../../rolling-shutter/bin/rolling-shutter
```
Comment (ylembachar) [inline play/manual/rolling-shutter:1]: this also would need to be updated as it's pointing to your local directory

File: play/manual/shuttermint.sh [hunk 7]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+(cd .. && bb init:testchain)
```
Comment (ylembachar) [inline play/manual/shuttermint.sh:3]: Could you also add steps to get the bb command running? thanks

File: play/manual/shuttermint.sh [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+(cd .. && bb chain) &
```
Comment (ylembachar) [inline play/manual/shuttermint.sh:4]: a couple additional steps we'd need to add: 
1. `install bb / go / abigen`
2. `export ROLLING_SHUTTER_BOOTSTRAP_SIGNING_KEY=479968ffa5ee4c84514a477a8f15f3db0413964fd4c20b08a55fed9fed790fad`
3. `export ROLLING_SHUTTER_CHAIN_GENESIS_KEYPER=0x440Dc6F164e9241F04d282215ceF2780cd0B755e`

**Pull request merged**